bcftools_1.15.1/bcftools_normvcf_1.15.1.cwl

```diff
@@ -0,0 +1,89 @@
+class: CommandLineTool
+cwlVersion: v1.0
+$namespaces:
+  dct: 'http://purl.org/dc/terms/'
+  doap: 'http://usefulinc.com/ns/doap#'
+  foaf: 'http://xmlns.com/foaf/0.1/'
+  sbg: 'https://www.sevenbridges.com/'
+id: bcftools_normvcf
+baseCommand:
+  - bcftools
+  - norm
+inputs:
+  - 'sbg:toolDefaultValue': s
```

Comment:
dont use any sbg tool definitions.  anything that is specific to sbg for the main tool we should not use